astropy/wcs/wcs.py

```diff
@@ -600,6 +600,7 @@ def calc_footprint(self, header=None, undistort=True, axes=None, center=True):
         Parameters
         ----------
         header : `~astropy.io.fits.Header` object, optional
+            Used to get ``NAXIS1`` and ``NAXIS2``
```

Comment:
Although reading over the original issue again I see that this was specifically requested.  I guess that's fine then but it should still be clarified what this is the header *of*.  

I think it should also be made clearer that this option and the `axes` option are mutually exclusive (the `header` argument is really just a shortcut for `axes=(header['NAXIS1'], header['NAXIS2'])`--it's just an alternative way to provide the same information.